compressors/fixed_bitwidth_compressor.py

```diff
@@ -161,7 +185,23 @@ def test_fixed_bitwidth_encode_decode():
     assert codelen == len(data_list) * 2 + alphabet_bits
 
 
-def test_fixed_bitwidth_file_encode_decode():
+def test_fixed_bitwidth_encode_decode():
```

Comment:
test also non-character alphabet?

Reply:
as in? can you elaborate?

Reply:
Just meant test on integer or something.